netlify.toml

```diff
@@ -10,7 +10,4 @@ HUGO_VERSION = "0.88.1"
 HUGO_THEME = "repo"
 
 [context.production]
-command = "npm run build:production"
-
-[context.production.environment]
-HUGO_ENV = "production"
```

Comment:
FYI, we're using Hugo command defaults and the `-e` flag to set the build mode so we don't need this anymore.